### PR TITLE
Add recipe for unidiff

### PR DESCRIPTION
**Pull request by tovrstra**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by tovrstra** [comment]
@conda-forge/help-python Ready for review. All seems fine to me. Unit tests included in recipe test section.

File: recipes/unidiff/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - unidiff
+  requires:
+    - python
```
Comment (beckermr) [inline recipes/unidiff/meta.yaml]: Not needed. Anything in the run requirements will get pulled in just fine.

File: recipes/unidiff/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+{% set name = "unidiff" %}
+{% set version = "0.5.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/matiasb/python-{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 5d351e402191b10e6b5164e6ce40b683f5f63eaedccc9a17d3ed47bbe3873634
```
Comment (beckermr) [inline recipes/unidiff/meta.yaml]: We are better off pulling from the pypi source https://pypi.org/project/unidiff/.

File: recipes/unidiff/meta.yaml (hunk 1 above)
Comment (tovrstra) [reply recipes/unidiff/meta.yaml]: good. just a second.

File: recipes/unidiff/meta.yaml (hunk 2 above)
Comment (tovrstra) [reply recipes/unidiff/meta.yaml]: The pypi sources do not contain unit tests, which is probably a bug, but still it makes the pypi source not as useful for this package. I'll open an issue on the unidiff repo.

Comment (tovrstra) [reply recipes/unidiff/meta.yaml]: Just made a PR: https://github.com/matiasb/python-unidiff/pull/61 PRs don't get merged quickly on this repo, so I'd rather not wait for it if you don't mind. Thanks for reviewing.

Comment (beckermr) [reply recipes/unidiff/meta.yaml]: It is ok for now. I don't consider running unit tests in the conda build needed. We are testing if a python package is installed properly here. An import does the trick. If the package does not work, that is an issue for the upstream package.

**Pull request merged**